### DISCUSSION TITLE
Custom runner labels in ARC

### DISCUSSION THREAD

**Discussion by dnyaneshwar2023** [actions/actions-runner-controller#2796] [General]
I want to use custom labels other than self-hosted.
Github Actions docs suggests something else like runner-scale-set.
I am using ARC controller + RunnerSet right now.
Is there any way to use custom labels so that I can use them in my workflow files?

**Comment by Link-** [comment]
Autoscaling runner set mode does not (and will not) support labels.

https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/about-actions-runner-controller#using-arc-runners-in-a-workflow

**Reply by ben-foxmoore** [reply]
@Link- can you share the thinking behind not supporting labels? They were a nice, flexible way to select runners, with a degree of separation between a team that maintains their own CI workflows, and the team maintaining the Runners.

**Reply by Link-** [reply]
Please check the answer here: https://github.com/actions/actions-runner-controller/discussions/2921

**Comment by Link-** [comment]
Locking this as a duplicate of: https://github.com/actions/actions-runner-controller/discussions/2921 - the conversation can continue there
